assets/src/components/inactiveNotificationModal.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useContext } from "react"
+import { StateDispatchContext } from "../contexts/stateDispatchContext"
+import { closeIcon } from "../helpers/icon"
+import { Notification } from "../realtime.d"
+import { setNotification } from "../state"
+import { title } from "./notifications"
+
+const InactiveNotificationModal = ({
+  notification,
+  removeNotification,
+  setInactiveNotification,
+}: {
+  notification: Notification
+  removeNotification: (id: number) => void
+  setInactiveNotification: (notification: Notification | null) => void
+}) => {
+  const [, dispatch] = useContext(StateDispatchContext)
+
+  const closeModal = () => {
+    setInactiveNotification(null)
+    dispatch(setNotification(undefined))
+  }
+
+  const removeNotificationAndClose = () => {
+    removeNotification(notification.id)
+    closeModal()
+  }
+
+  return (
+    <>
+      <div className="m-modal">
+        <div className="m-invalid-notification-modal__close-button">
+          <button onClick={closeModal}>{closeIcon()}</button>
+        </div>
+        <div className="m-notification__title">
+          {title(notification.reason)} NOTIFICATION
+        </div>
+        <div className="m-invalid-notification-modal__body">
+          {runIdPhrase(notification)} currently active in Skate. This
+          notification may no longer be relevant.
+        </div>
+        <div className="m-invalid-notification-modal__buttons">
+          <div className="m-invalid-notification-modal__button-wrapper">
+            <button
+              className="m-invalid-notification-modal__keep-button"
+              onClick={closeModal}
+            >
+              Keep
+            </button>
+          </div>
+          <div className="m-invalid-notification-modal__button-wrapper">
+            <button
+              className="m-invalid-notification-modal__discard-button"
+              onClick={removeNotificationAndClose}
+            >
+              Remove
+            </button>
+          </div>
+        </div>
+      </div>
+      <div className="m-modal-overlay" />
+    </>
+  )
+}
+
+const runIdPhrase = (notification: Notification): string => {
+  if (notification.runIds.length === 0) {
+    return "No runs associated with this notification are"
+  }
+
+  if (notification.runIds.length === 1) {
+    return `Run ${notification.runIds[0]} is not`
```

Comment:
Design question about a weird edge case that (if we decide to act on it) would be a separate PR after the release: @ingridpierre 

It's mostly but not entirely true that the run isn't active. The block waiver might not be applied to the whole run, and if a bus is on the right run, but doing a trip that the block waiver wasn't applied to, we wouldn't match it. Should we:
1. Change the wording to something to the effect of "Nobody's doing the trips this notification applies to"
2. Change the matching so we can show a bus that's doing any part of this run, not just the trips on the block waiver
3. Leave it as is

?